transform/README.md

```diff
@@ -13,3 +13,5 @@ Try running the following commands:
 - Join the [chat](https://community.getdbt.com/) on Slack for live discussions and support
 - Find [dbt events](https://events.getdbt.com) near you
 - Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+
+![dbt Lineage Diagram](docs/dbt_dag.png)
```

Comment:
Suggestion: Remove this README file completely as it's boilerplate and also redundant with the readme file in the main folder. Wdyt?

Reply:
Oh yeah, you are right. Thanks for spotting that out.